python/ray/autoscaler/azure/config.py

```diff
@@ -59,169 +57,50 @@ def _configure_resource_group(config):
     resource_client.resource_groups.create_or_update(
         resource_group_name=resource_group, parameters=params)
 
-    return config
-
-
-def _configure_msi_user(config):
-    msi_client = _get_client(ManagedServiceIdentityClient, config)
-    resource_client = _get_client(ResourceManagementClient, config)
-    auth_client = _get_client(AuthorizationManagementClient, config)
+    # load the template
+    template_path = os.path.join(
+        os.path.dirname(__file__), 'azure-config-template.json')
+    with open(template_path, 'r') as template_file_fd:
+        template = json.load(template_file_fd)
+
+    # choose a random subnet
+    random.seed(resource_group)
+    # start at 1 to avoid most likely collision at 0
+    parameters = {"subnet": "10.{}.0.0/16".format(random.randint(1, 254))}
+
+    deployment_properties = {
+        'mode': DeploymentMode.incremental,
```

Comment:
this should be "Incremental" ?

Reply:
well, if the previous deployment fails, this allows you to resume...

Reply:
I mean that DeploymentMode isn't a valid object

Reply:
oh i see, need `from azure.mgmt.resource.resources.models import DeploymentMode`